### DISCUSSION TITLE
help fatal error database

### DISCUSSION THREAD

**Discussion by Simonk007** [inventree/InvenTree#2470] [General]
Hi,
When runing docker-compose up i get an error, what am i misisng

 inventree docker-compose up --force-recreate
Recreating inventree-db ... done
Recreating inventree-server ... done
Recreating inventree-worker ... done
Recreating inventree-proxy  ... done
Attaching to inventree-db, inventree-server, inventree-worker, inventree-proxy
inventree-proxy     | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
inventree-proxy     | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
inventree-db        |
inventree-db        | PostgreSQL Database directory appears to contain a database; Skipping initialization
inventree-db        |
inventree-db        | 2021-12-20 20:57:56.757 UTC [1] LOG:  starting PostgreSQL 13.5 (Debian 13.5-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
inventree-db        | 2021-12-20 20:57:56.757 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
inventree-db        | 2021-12-20 20:57:56.757 UTC [1] LOG:  listening on IPv6 address "::", port 5432
inventree-db        | 2021-12-20 20:57:56.763 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
inventree-db        | 2021-12-20 20:57:56.769 UTC [29] LOG:  database system was shut down at 2021-12-20 20:57:48 UTC
inventree-db        | 2021-12-20 20:57:56.775 UTC [1] LOG:  database system is ready to accept connections
inventree-proxy     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
inventree-server    | /home/inventree/data/config.yaml exists - skipping
inventree-server    | [2021-12-20 20:57:58 +0000] [1] [INFO] Starting gunicorn 20.1.0
inventree-server    | [2021-12-20 20:57:58 +0000] [1] [INFO] Listening at: http://0.0.0.0:8000 (1)
inventree-server    | [2021-12-20 20:57:58 +0000] [1] [INFO] Using worker: sync
inventree-server    | [2021-12-20 20:57:58 +0000] [9] [INFO] Booting worker with pid: 9
inventree-server    | [2021-12-20 20:57:58 +0000] [10] [INFO] Booting worker with pid: 10
inventree-server    | [2021-12-20 20:57:58 +0000] [11] [INFO] Booting worker with pid: 11
inventree-server    | [2021-12-20 20:57:58 +0000] [12] [INFO] Booting worker with pid: 12
inventree-worker    | /home/inventree/data/config.yaml exists - skipping
inventree-proxy     | 10-listen-on-ipv6-by-default.sh: info: can not modify /etc/nginx/conf.d/default.conf (read-only file system?)
inventree-proxy     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
inventree-proxy     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
inventree-proxy     | /docker-entrypoint.sh: Configuration complete; ready for start up
inventree-proxy     | 2021/12/20 20:57:58 [notice] 1#1: using the "epoll" event method
inventree-proxy     | 2021/12/20 20:57:58 [notice] 1#1: nginx/1.20.2
inventree-proxy     | 2021/12/20 20:57:58 [notice] 1#1: built by gcc 10.2.1 20210110 (Debian 10.2.1-6)
inventree-proxy     | 2021/12/20 20:57:58 [notice] 1#1: OS: Linux 5.4.0-84-generic
inventree-proxy     | 2021/12/20 20:57:58 [notice] 1#1: getrlimit(RLIMIT_NOFILE): 1048576:1048576
inventree-proxy     | 2021/12/20 20:57:58 [notice] 1#1: start worker processes
inventree-proxy     | 2021/12/20 20:57:58 [notice] 1#1: start worker process 23
inventree-proxy     | 2021/12/20 20:57:58 [notice] 1#1: start worker process 24
inventree-proxy     | 2021/12/20 20:57:58 [notice] 1#1: start worker process 25
inventree-db        | 2021-12-20 20:57:59.298 UTC [36] FATAL:  database "inventree" does not exist

**Comment by SchrodingersGat** [comment]
Did you follow the instructions at https://inventree.readthedocs.io/en/latest/start/docker_prod/

In particular, the step where you create the database: https://inventree.readthedocs.io/en/latest/start/docker_prod/#create-database

**Reply by Simonk007** [reply]
sorry didnt see that, i will give it a try

**Reply by eeintech** [reply]
@Simonk007 Please mark it as answer when you have to chance :smiley:

**Reply by Simonk007** [reply]
how do i mark it

**Reply by eeintech** [reply]
Nevermind this is not a QA discussion
